paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -765,7 +765,27 @@ class PaparazziPluginTest {
     assertThat(resourceFileContents[4]).isEqualTo("intermediates/assets/debug")
     assertThat(resourceFileContents[5]).isEqualTo("app.cash.paparazzi.plugin.test,com.example.mylibrary")
     assertThat(resourceFileContents[6]).isEqualTo("src/main/res,src/debug/res")
-    assertThat(resourceFileContents[7]).matches("^caches/transforms-3/[0-9a-f]{32}/transformed/external/res\$")
+    assertThat(resourceFileContents[8]).matches("^caches/transforms-3/[0-9a-f]{32}/transformed/external/res\$")
+  }
+
+  @Test
+  fun verifyResourcesGeneratedForMultiModuleProject() {
+    val fixtureRoot = File("src/test/projects/verify-resources-modules")
+
+    gradleRunner
+      .withArguments("compileDebugUnitTestKotlin", "--stacktrace")
+      .runFixture(fixtureRoot) { build() }
+
+    val resourcesFile = File(fixtureRoot, "build/intermediates/paparazzi/debug/resources.txt")
+    assertThat(resourcesFile.exists()).isTrue()
+
+    val resourceFileContents = resourcesFile.readLines()
+    assertThat(resourceFileContents[0]).isEqualTo("app.cash.paparazzi.plugin.test")
+    assertThat(resourceFileContents[1]).isEqualTo("intermediates/merged_res/debug")
+    assertThat(resourceFileContents[4]).isEqualTo("intermediates/assets/debug")
+    assertThat(resourceFileContents[5]).isEqualTo("app.cash.paparazzi.plugin.test")
+    assertThat(resourceFileContents[6]).isEqualTo("src/main/res,src/debug/res")
+    assertThat(resourceFileContents[7]).isEqualTo("module1/build/intermediates/packaged_res/debug,module2/build/intermediates/packaged_res/debug")
```

Comment:
can we merge this test into the original, instead of having two separate tests testing different line number in resources.txt?